cmd/qtalk/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/progrium/clon-go"
+	"github.com/progrium/qtalk-go/codec"
+	"github.com/progrium/qtalk-go/talk"
+)
+
+func main() {
+	flag.Parse()
+
+	cmd := flag.Arg(0)
+	if cmd != "call" {
+		log.Fatal("unknown command")
+		return
+	}
+
+	u, err := url.Parse(flag.Arg(1))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var args any
+	if len(flag.Args()) > 2 {
+		args, err = clon.Parse(flag.Args()[2:])
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	peer, err := talk.Dial(u.Scheme, u.Host, codec.JSONCodec{})
```

Comment:
I assume this should still have a `peer.Close()` call to cleanly shut down the connection, right?

Reply:
Good catch